sympy/core/tests/test_numbers.py

```diff
@@ -849,7 +849,7 @@ def test_Infinity_inequations():
     x = Symbol('x')
     b = Symbol('b', finite=True, real=True)
     assert (x < oo) == Lt(x, oo)  # issue 7775
-    assert b < oo and b > -oo and b <= oo and b >= -oo
+    assert oo > b > -oo and b <= oo and b >= -oo
```

Comment:
Same here, these tests may be explicitly testing the inequality as written. I would be more cautious about changing the tests, especially tests for inequalities themselves.